### PR TITLE
WIP update on developer docs

### PR DESCRIPTION
**Pull request by oscgonfer**
Hi @timcowlishaw 

This is now pretty advanced I think. Can you check the Devices part and potentially the errors file? The world map bit is also waiting until we decide on the trimming!

### CODE REVIEW

File: source/includes/_filtering_responses.md.erb [hunk 1]
```diff
@@ -66,48 +66,51 @@ Similar to the [pagination](#pagination), you can filter and sort most responses
...
 uuid |
 
+### Examples
+
+Query | Type | Value | Description
+--------- | ------- | ------- | -----------
+/users?q[**username_eq**]=adam | string | adam | Users where first name equals 'Adam'
+/devices?q[**owner_id_eq**]=1 | integer | 1 | Devices where owner ID equals 1
+/devices?q[**tags_name_in**]=Barcelona | string | Barcelona |  Devices which have the tag 'Barcelona' (case sensitive).
+/devices?**with_tags**=Barcelona&#124;Amsterdam | string | Barcelona&#124;Amsterdam | Devices which have ANY of the tags separated by PIPE &#124;
```
Comment (timcowlishaw) [inline source/includes/_filtering_responses.md.erb:109]: In API v1 we can remove this, see: https://github.com/fablabbcn/smartcitizen-api/issues/186

Comment (timcowlishaw) [reply source/includes/_filtering_responses.md.erb:109]: Might be worth "deprecating" it in documentation now, and specifying that **tags_name_in** can be specified multiple times?

File: source/includes/_forwarding.md.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Data forwarding
+
+<aside class='notice'>This is an experimental feature. [Get in touch with us](mailto:support@smartcitizen.me) for more information, as it may change.</aside>
+
+## MQTT
+
+Instead using **websockets**, you can also use MQTT, to get _rendered_ data from the devices in real-time. This can help into bringing MQTT features such as persistent sessions and allow for a more robust data ingestion. We support MQTT/s, MQTT over SSL/TLS, like HTTPS. Everytime a device publishes data to the platform data is pushed over, with the same model as <a href="/#get-a-device">device `GET`</a>.
```
Comment (timcowlishaw) [inline source/includes/_forwarding.md.erb]: Instead **of** using websockets

**Review by timcowlishaw — APPROVED** [review]
This looks good Oscar! a couple of small comments but overall :+1:

File: source/includes/_forwarding.md.erb (hunk 2 above)
Comment (oscgonfer) [reply source/includes/_forwarding.md.erb]: Thanks!

**Pull request merged**